virtwho/virt/ahv/ahv.py

```diff
@@ -276,11 +276,16 @@ def _validate_server(self, key):
             Socket error is returned in case of an invalid ip.
         """
         error = super(AhvConfigSection, self)._validate_server(key)
-        try:
-            ip = self._values[key]
-            socket.inet_aton(ip)
-        except socket.error:
-            error = 'Invalid server IP address provided'
+        if error is None:
```

Comment:
Any chance this could end up being an empty list or empty tuple at this point (anything else false-like but not None)?

Reply:
From looking more at the VirtConfigSection._validate_server() method, it seems checking for None only works.

Reply:
it is consistent with the other _validate_server override methods.